src/components/story/Button.tsx

```diff
@@ -0,0 +1,21 @@
+interface Props {
+  text: string;
+  redirect: string;

```

Comment:
I think it would be better if we used `handleClick` as props, as it would provide more flexibility for the button.